addon/index.js

```diff
@@ -1,54 +1,14 @@
 import { task as ecTask } from 'ember-concurrency';
+import { decorator } from '@ember-decorators/utils/decorator-wrappers';
+import extractValue from '@ember-decorators/utils/extract-value';
 
-function extractValue(desc) {
-  return desc.value ||
-    (typeof desc.initializer === 'function' && desc.initializer());
-}
-
-function isDescriptor(item) {
-  return item &&
-    typeof item === 'object' &&
-    'writable' in item &&
-    'enumerable' in item &&
-    'configurable' in item;
-}
-
-function handleDescriptor(callback, target, key, desc, params = []) {
-  return {
-    enumerable: desc.enumerable,
-    configurable: desc.configurable,
-    writeable: desc.writeable,
-    initializer: function() {
-      if (!desc.writable) {
-        throw new Error('ember-concurrency-decorators does not support using getters and setters');
-      }
```

Comment:
ah dang...forgot about this.  probably need to make sure this is still in there....